src/networkremote/incomingmsg.cpp

```diff
@@ -0,0 +1,43 @@
+#include "incomingmsg.h"
```

Comment:
Wrong header name, should match the class name in lowercase.

Reply:
Do you want me to rename all my files to match the class?